packages/trpc/server/routers/viewer/organizations/updateUser.handler.ts

```diff
@@ -74,30 +79,58 @@ export const updateUserHandler = async ({ ctx, input }: UpdateUserOptions) => {
           },
         },
       },
+      user: {
+        select: {
+          username: true,
+        },
+      },
     },
   });
 
   if (!requestedMember)
     throw new TRPCError({ code: "UNAUTHORIZED", message: "User does not belong to your organization" });
 
-  let avatar = input.avatar;
-  if (input.avatar) {
-    avatar = await resizeBase64Image(input.avatar);
+  const hasUsernameUpdated = input.username !== requestedMember.user.username;
+
+  if (input.username && hasUsernameUpdated && user.profile.organization?.slug) {
+    const checkRegularUsernameRes = await checkRegularUsername(
```

Comment:
NIT: I noticed that `checkRegularUsername` is used only within an org context. Should we rename this function so it's more obvious when reading the function name?